CorsixTH/Lua/dialogs/confirm_dialog.lua

```diff
@@ -58,7 +58,7 @@ function UIConfirmDialog:UIConfirmDialog(ui, text, callback_ok, callback_cancel)
   local y = topFrameHeight;
   for i=1, totalLines do
     self:addPanel(358, 0, y)  -- Dialog background
-    y = y + textRowHeight
+    y = topFrameHeight + (i * textRowHeight)
```

Comment:
Unused variable names can be replaced with `_` and then it will ignore them. That's better than making this more complicated.

Reply:
Agreed - have just checked in.